app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -184,6 +184,17 @@
       display: block;
     }
   }
+
+  // position above alert when alert is present and viewport > breakpoint
+  .global-alert + & {
+    margin-top: -4.5rem;
+    margin-bottom: 4.5rem;
+
```

Comment:
Nice use of the adjacent selector, I always forget about this.

But I don't think it's a good idea using an absolute value for the global alert size. I found an issue when you sign up, sometimes the message wraps onto two lines:

![image](https://cloud.githubusercontent.com/assets/14920201/13048485/0d5beaf4-d3de-11e5-9987-d97e7245e69b.png)